app/src/main/res/xml/grayscale_icon_map.xml

```diff
@@ -76,7 +76,8 @@
     <icon drawable="@drawable/gallery" package="com.sec.android.gallery3d" name="Gallery" />
     <icon drawable="@drawable/gallery" package="com.oneplus.gallery" name="Gallery" />
     <icon drawable="@drawable/gallery" package="com.miui.gallery" name="Gallery" />
-    <icon drawable="@drawable/browser" package="com.sec.android.app.sbrowser" name="Browser" />
+    <icon drawable="@drawable/browser" package="org.droidtr.jelly" name="Browser" />
+    <icon drawable="@drawable/sbrowser" package="com.sec.android.app.sbrowser" name="Browser" />
```

Comment:
The app is listed on Google Play under the name Samsung Internet Browser, so I’d suggest updating the value of the `name` attribute accordingly.

```suggestion
    <icon drawable="@drawable/sbrowser" package="com.sec.android.app.sbrowser" name="Samsung Internet Browser" />
```